alot/ui.py

```diff
@@ -145,6 +145,10 @@ def _error_handler(self, exception):
             self.notify(str(exception), priority='error')
         elif isinstance(exception, CommandCanceled):
             self.notify("operation cancelled", priority='error')
+        elif isinstance(exception, SequenceCanceled):
```

Comment:
just to make sure I understand: Without introducing this new exception, and raising `CommandCanceled` instead, would only abort the current Command but not the whole sequence?

Reply:
Yes. That's what I tried initially, but CommandCanceled gets handled in self.apply_command and not re-raised, so never trickles up to self.apply_commandline, so a normal CommandCanceled in a sequence doesn't cancel the sequence.

Reply:
Ok thanks for clarifying.
IIRC, CommandCanceled was there to enable confirmation prompts inside a single command, such as when sending a message.
I'm not sure if we really need two different types of cancellation exceptions. I mean, it seems to me that all other use cases would be fine if the whole *sequence* of commands is killed. This would only be a problem if you chain commands with prompts, and *really* want to continue despite having negatively answered some prompt. useful?

Reply:
I agree that that doesn't sound very useful. We could take the CommandCanceled handling out of the unified exception handler function and handle it differently in the exception handlers of apply_command and apply_commandline. I didn't want to mess too much with the unified exception handling function when initially creating this.